src/praisonai-agents/tests/unit/test_delivery_target_preview.py

```
@@ -0,0 +1,68 @@
+
+def test_delivery_validation_ok_defaults():
+    v = DeliveryValidation(ok=True, preview="telegram:@alice")
+    assert v.ok is True
+    assert v.reason == ""
+    assert v.hint == ""
+    assert v.preview == "telegram:@alice"
+
+
+def test_delivery_validation_frozen():
+    v = DeliveryValidation(ok=True)
+    with pytest.raises(Exception):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)